app/src/main/kotlin/com/hedvig/app/feature/chat/ui/ChatActivity.kt

```diff
@@ -76,8 +76,6 @@ class ChatActivity : BaseActivity(R.layout.activity_chat) {
 
   private var forceScrollToBottom = true
 
-  override val preventRecreation = true
```

Comment:
Since this isn't really gonna be supported anymore I wonder, what was the use case in the first place?
From what I understand some screens were done in a way where we did not want to restart on config change triggered from locale change for whatever reason. If this reason is still something we care about do we need to do something special about it?

Reply:
To be honest, I really don't remember. 🤔

Reply:
Haha that's also a valid response. In which case I'd say since I don't think we're actively using this let's just let it get deleted and observe if anything breaks 👀